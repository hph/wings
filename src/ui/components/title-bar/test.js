import React from 'react';
import renderer from 'react-test-renderer';

import TitleBar from './index';

const defaultProps = {
  label: '',
};

const createSnapshot = (props = defaultProps) => {
  expect(
    renderer.create(<TitleBar {...props} />).toJSON(),
  ).toMatchSnapshot();
};

test('TitleBar renders a simple div with a class', () => {
  createSnapshot();
});

test('TitleBar renders a label inside it', () => {
  createSnapshot({ label: 'Wings is great!' });
});
