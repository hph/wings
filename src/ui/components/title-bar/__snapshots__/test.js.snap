// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBar renders a label inside it 1`] = `
<div
  className="root"
>
  Wings is great!
</div>
`;

exports[`TitleBar renders a simple div with a class 1`] = `
<div
  className="root"
>
  
</div>
`;
